src/Plugins/LevelDBStore/Plugins/Storage/Snapshot.cs

```diff
@@ -48,7 +48,8 @@ public void Dispose()
 
         public IEnumerable<(byte[] Key, byte[] Value)> Seek(byte[] prefix, SeekDirection direction = SeekDirection.Forward)
         {
-            return db.Seek(options, prefix, direction, (k, v) => (k, v));
+            return db.Seek(options, prefix, direction, (k, v) => (k, v))
```

Comment:
Could you explain what are you trying to fix?

Reply:
Im guessing, so the `snapshot` can read it items that are added.

Reply:
> Could you explain what are you trying to fix?

Reverted that change, cause in neo core, we can read added item in the snapshot, but here in the store the logic is different. Checked again, what we called snapshot in the core is actually snapshot cache.